module-apps/apps-common/locks/handlers/PhoneLockHandler.cpp

```diff
@@ -486,11 +486,16 @@ namespace locks
         phoneLockTimer.start();
         phoneLockTimeUpdateInfoAction(LockTimerState::Start);
 
+        setNextUnlockAttemptFormattedTime();
+    }
+
+    void PhoneLockHandler::setNextUnlockAttemptFormattedTime() noexcept
+    {
         if (lockedFor < utils::secondsInMinute) {
             lock.setNextUnlockAttemptFormattedTime(std::to_string(lockedFor) +
                                                    utils::translate("phone_lock_blocked_information_seconds"));
```

Comment:
is the formatting right there? :D

Reply:
That's how it was formatted automatically, so I suppose it should be correct